### ISSUE TITLE
Missing Test Set

### ISSUE THREAD

**Issue by jhkonan** [NNNNAI/VGGFace2-HQ#8]
I unzipped the files on Google Drive, but it only creates a directory with 8631 classes for the training set. Is it possible for you to add the remaining 500 classes for VGGFace2 test set?

Thank you for sharing this resource!

**Comment by NNNNAI** [comment]
Thanks for your attention to our work.You can used the following command line.I only generated HQ version of VGGFACE2 training set. I will upload the hq version of vggface2 testing set when I got time to do that. Maybe you can try to generate it by following the steps in readme.md.

**Comment by jhkonan** [comment]
Thank you NNNNAI, if you could upload the test set -- that would be great. Getting the repo running on the original data from scratch is proving to be a challenge... we also want to reference this repo as the data source vs a processing resource.

Do you know when you might have time to make this upload?

Thank you again.

**Comment by NNNNAI** [comment]
Cause I did got the vggface2 testing set right now, maybe I will upload the HQ version of the test set after I download the test set.

**Comment by jhkonan** [comment]
Should we expect the test set to be on Google Drive or Baidu Drive? I just want to make sure I am checking the right place.

Thank you for taking the time to make this addition.

**Comment by jhkonan** [comment]
Are there any updates on the test set?

**Comment by jhkonan** [comment]
Sorry, I hate to bother you. Can you please upload the test set?

**Comment by zimenglan-sysu-512** [comment]
Are there any updates on the test set?
